lib/resty/redis.lua

```diff
@@ -38,7 +38,7 @@ local common_cmds = {
     "zrange",   "zrangebyscore", "zrank",   "zadd",
     "zrem",     "zincrby",                              -- Sorted Sets
     "auth",     "eval",         "expire",   "script",
-    "sort"                                              -- Others
+    "sort", "ping", "time"                              -- Others
```

Comment:
We only put truly common commands in this fast table for performance. Other commands can be loaded lazily anyway, including any feature-added redis commands.